packages/sampler/stories/qa/Scroller.js

```diff
@@ -69,6 +69,36 @@ class ScrollerWithResizable extends React.Component {
 	}
 }
 
+class ScrollerLastFocusedItem extends React.Component {
```

Comment:
This sample doesn't focus last focused, but 'top most visible item in the scroller'.  We should update names and titles to reflect that.